### ISSUE TITLE
Uninstall of LibreOffice trashed Gecko install 

### ISSUE THREAD

**Issue by DPinNM** [geckolinux/geckolinux-project#124]
New Gecko user and still fairly new to Linux. Have experience with about 6 other distros. I would never have been tempted by OpenSUSE but when I read that it a) worked with all kinds of hardware ( found that both Manjaro and Ubuntu did not play nicely with the Broadcom BCM-4313 wireless adapter I have in 2 of my 3 laptops) and then b) learned of Gecko Linux as a much friendlier front-end to OpenSUSE, I thought I'd give it a try. It's been on an Acer Aspire One 722 for a bit more than a week and had been working fine...including no problem with the BCM-4313.

Because I don't use LibreOffice (too incompatible with MS Word) it's just excess baggage and so today I wanted to remove/uninstall it. With other distros this has been as easy as right-click/choose Uninstall...and it's never been more complicated than removing using Synaptic. 

I used System > Install/Remove Software and entered LibreOffice. When all of the many dependencies that were automatically selected had been removed and a ton of stuff installed (a process that took a very long time) a restart of Gecko showed the whole install was trash. I've re-installed it and am in the process of adding the few packages I prefer (sure wish Gecko/OpenSUSE had Geary without all of the bad choices for installing it) and I'd like to know what I overlooked or did wrong.  Obviously, I could just leave LibreOffice alone and use the FreeOffice I prefer...but I'd really prefer to get rid of packages I don't use when they are that big.  Thanks very much!

**Comment by lasofeli** [comment]
Not the project maintainer/developer but there shouldn't be things installing when you're really just uninstalling a bunch of packages. Maybe you accidentally (or knowingly) triggered a system update. If that's the case, can you share if you're using the rolling or stable version of Gecko? Afaik, in the Rolling version or OpenSUSE Tumbleweed, you're not supposed to use the YaST tool to update your system. You're supposed to use the terminal command zypper dup

**Comment by DPinNM** [comment]
Thanks for the reply. I can't help but note that it took 27 days for someone to reply here and, when that reply finally came, it wasn't from the developer. So.....a bit of a ghost town here, eh?

So, yeah, I did do the update via sudo zypper dup but only after I performed a fresh install after the LibreOffice fiasco.  However, as Blaine Packer (InfinitelyGalactic) noted on his YouTube channel review, the .iso dates from the middle of 2018!  The update consists of 1700+ packages!  My first attempt at it failed (only after waiting for everything...and then it was the final scripting that didn't write)...but I was luckier on the second try. But because I did this with the specific intention of getting the updates, that's why I don't think I accidentally triggered an update when all I wanted to do was uninstall LibreOffice. With some distros, uninstalling packages is as easy as a right-click from the menu. With others....it's much more fuss.
